examples/auto-relay/relay.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const Libp2p = require('libp2p')
+const Websockets = require('libp2p-websockets')
+const { NOISE } = require('libp2p-noise')
+const MPLEX = require('libp2p-mplex')
+
+;(async () => {
+  const node = await Libp2p.create({
+    modules: {
+      transport: [Websockets],
+      connEncryption: [NOISE],
+      streamMuxer: [MPLEX]
+    },
+    addresses: {
+      listen: ['/ip4/0.0.0.0/tcp/0/ws']
+      // announceFilter: TODO check production section
```

Comment:
What production section?

Reply:
Changed, it was the `What's next section` where I mention that the announceAddrs should be changed for production